.rubocop.yml

```diff
@@ -519,5 +519,26 @@ RSpec/RepeatedDescription:
 RSpec/NestedGroups:
   Enabled: False
 
+# disable Yaml safe_load. This is needed to support ruby2.0.0 development envs
 Security/YAMLLoad:
-  Enabled: False
+  Enabled: false
+
+
+#------------------------------
+# CUSTOM 
+
+# Offense count: 2
+# Configuration parameters: AssignmentOnly.
+RSpec/InstanceVariable:
```

Comment:
there is more overriden next modulesync. all in https://github.com/voxpupuli/puppet-puppetserver/pull/40/commits/f711d505a9b5c51e2ecfcfefd15f63ed037c8b76

if you know a better way just fix it yourself. :-)